lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -396,9 +396,11 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
                 callback, codec.encodeSuccessEnvelope(true));
             return;
           case 'SystemChrome.setApplicationSwitcherDescription':
-            final Map<String, dynamic> arguments = decoded.arguments;
-            domRenderer.setTitle(arguments['label']);
-            domRenderer.setThemeColor(ui.Color(arguments['primaryColor']));
+            final Map<String, Object> arguments = decoded.arguments;
+            final String label = arguments['label'] as String? ?? '';
+            final int primaryColor = arguments['primaryColor'] as int? ?? 0;
```

Comment:
Are there any breadcrumbs we could leave so we know where these defaults are coming from? For example, in other places we link to the C++ variant engine code and advise that people keep the values in sync.

Reply:
Let me check what the mobile engine does, and I'll reproduce those values there. I'll comment to the relevant implementation detail too.

Reply:
[iOS does nothing](https://github.com/flutter/engine/blob/138c91c614d742c52aa5432b4cb921f0ff9fdee2/shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm#L155-L157), and android [passes values through](https://github.com/flutter/engine/blob/138c91c614d742c52aa5432b4cb921f0ff9fdee2/shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java#L185-L205).

The only documentation I found about this, in the Android docs, is that the primaryColor *must be opaque*, [here](https://developer.android.com/reference/kotlin/android/app/ActivityManager.TaskDescription#TaskDescription(kotlin.String,%20kotlin.Int,%20kotlin.Int)). This is used in Android to customize the name and color of the app [when task-switching](https://stackoverflow.com/questions/44682593/how-do-i-style-the-window-bar-when-my-android-app-is-in-background).

I'll make this an opaque black when null, as you suggested: `0xFF000000`

Reply:
Android seems to ignore `null` primary color. [Source](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/app/ActivityManager.java#1257). It falls-back to [default `ActivityTaskDescription_colorPrimary`](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/app/Activity.java#5100), which I guess is defined in some App theme (being in R)